lib/charms/bind/v0/dns_record.py

```diff
@@ -268,7 +266,8 @@ def validate_dns_entry(self, _: ValidationInfo) -> "RequirerEntry":
             the DNS entry if valid.
         """
         validated_entry = RequirerEntry.model_validate(self)
-        # Additional validations will be done here
+        # Additional validations will be done here in the form of assertions
+        # assert validated_entry.domain == "cloud.canonical.com"
```

Comment:
Would this comment be intended?

Reply:
Yes, for now. It will be removed later on